### DISCUSSION TITLE
permission denied for table http_request_queue

### DISCUSSION THREAD

**Discussion by letahitien** [supabase/supabase#20854] [Questions]
Hi, I've created a webhook which is pointed to a public table and which uses the DELETE event.

Normally, when I delete a row in this table, the webhook is called to execute the operation. But I'm getting this error: "permission denied for table http_request_queue".

I need your help, I don't know what to do, thank you.

**Comment by letahitien** [comment]
I took a screenshot of the problem in question :

![Error](https://github.com/supabase/supabase/assets/45018350/9a06b984-3c63-4daa-b13f-93b6f31f022f)

**Reply by doriancollier** [reply]
I'm having similar issues after adding a webhook. Did you find a solution?

**Comment by sjuery** [comment]
Also running into this issue
